sim/constants.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Simulator constants
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * @license MIT
+ */
+
+const k = {
+	EFFECTIVENESS: {
+		INVALID: 'na',
+		NEUTRAL: 'e',
+		NO_EFFECT: 'ne',
+		NOT_VERY: 'nve',
+		SUPER: 'se',
+	},
+	MV_RESULT: {
```

Comment:
These three should be the only constants on `Battle`, and they should be directly on `Battle` imo.